documentdb.tf

```diff
@@ -0,0 +1,43 @@
+module "documentdb-cluster" {
+  providers = {
+    aws = aws.provisionaccount
+  }
+
+  source  = "cloudposse/documentdb-cluster/aws"
+  version = "0.27.0"
+
+  cluster_size    = var.db_cluster_size
+  db_port         = var.db_port
+  instance_class  = var.db_instance_class
+  master_password = var.db_password
+  master_username = var.db_username
+  name            = var.db_name
+  namespace       = terraform.workspace # TODO: Decide if this should be a parameter
```

Comment:
Needs an issue unless this is resolved before the PR is taken out of draft mode.

Reply:
I'm going to punt on this for now and deal with it later when we decide if/how we are going to stop using Terraform workspaces.